CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+Changelog
+===
+
+### 3.12
+
+Release date: TBD, 2017
+
+* [JENKINS-40621](https://issues.jenkins-ci.org/browse/JENKINS-40621) - 
+Prevent leaked file descriptorswhen invoking `MavenEmbedderUtils#getMavenVersion()`
+([PR #5](https://github.com/jenkinsci/lib-jenkins-maven-embedder/pull/5))
+* Update to Guice `4.0-beta`
```

Comment:
See also 
https://issues.jenkins-ci.org/browse/JENKINS-38736
https://github.com/jenkinsci/maven-plugin/pull/87
We will have to bump to the released version
It can be done in another PR/commit